src/cart/cart.module.ts

```diff
@@ -3,11 +3,15 @@ import { Module } from '@nestjs/common';
 import { Cart, CartSchema } from './schema/cart.schema';
 import { CartService } from './service/cart.service';
 import { CartController } from './controller/cart.controller';
+import { Product, ProductSchema } from '~/products/schema/product.schema';
+import { Order, OrderSchema } from '~/order/schema/order.schema';
 
```

Comment:
I think we can reorder the imports... so we can have other modules first then, we can have other files come in after